server/lib/config/config.ts

```diff
@@ -1,11 +1,11 @@
 import * as fs from 'fs';
-import * as path from 'path';
+import *  as path from 'path';
 import convict from 'convict';
 import { ipaddress, url } from 'convict-format-with-validator';
 import json5 from 'json5';
 import yaml from 'yaml';
 import toml from 'toml';
-import { cpus, networkInterfaces } from 'os';
```

Comment:
Seems like after merging `develop` into your branch the @sd4v1d  's changes (new functionality: detecting ip) has been completely lost. Bring the `develop` version of this file back.